<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>iDEC NEFU_China 2021</title>
    <link rel="stylesheet" type="text/css" href="../../CSS/main_cssstyle.css">
    <style>
        header{
            background: url("../../resources/header1.png");
            min-height: 450px;
            background-repeat: no-repeat;
            background-size: cover;
        }
        .page-title {
            padding-top:10em;
            padding-bottom: 1.5em;
        }
        /* footer的样式 */
        .contact-icon{
            display: flex;
            align-items: center;
            /*font-size: 3em;*/
            color: white;
        }
        .contact-icon a{
            color: white;
        }
        .contact-icon i{
            font-size: 3em;
        }
    </style>
</head>
<body>
<!--加载动画-->
<div class="loader-wrapper">
    <span class="loader"><img src="../../resources/Loading.gif" alt="Our animated logo is keeping you company until the page has loaded.">
    </span>
</div>

<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
<header>
    <!------------ 页面标题PAGE TITLE -------------------->
    <div class="page-title">
        <div class="page-title-content">
            <h1>Engineering</h1><!---Page title -->
        </div>
    </div>
</header>
<div class="grid-normal">
    <!--------------Introduction/Summary--------------------->
    <div class="introduction grid-normal">
    </div>
</div>
<div style="display:none" id="marker" class= "triangle-down"> </div>
<!------------------ BUTTONS ---------------------------->
<div class="buttons">
</div>
<div class="sections-container">
    <aside class="sticky">
        <div id="sub-nav"class="sub-navigation-content">
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter1">Introduce</a><!------Chapter one -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter2">Design</a><!------Chapter Two -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter3">Build</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter4">Test</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter5">Learn</a><!------Chapter Three -->
            </div>
            <div class="sub-navigation-chapter cool-link2">
                <a class= "main-nav-link" linkedidchapter="slide1-chapter6">Outlook</a><!------Chapter Three -->
            </div>
        </div>
    </aside>
    <div class="carousel">
        <div class="slider">
            <!------------------------------ section 1------------------------------------>
            <section class="slide-sections">
                <div class="main-body">
                    <div class="main-body-content">
                        <div id="slide1-chapter1">
                            <h2>Introduce</h2>
                            <p>Welcome to the Engineering page, where we introduce our engineering concept based on synthetic biology, research progress and success in our project design. We also report the methods used by us to test the unbiasedness and feasibility of the concept.</p>
                            <p>As what we have indicated in the Description, our ultimate goal is to detect nucleic acid of viruses. A genuine nucleic acid detection system should have the following characters: sensitivity, specificity, and convenience. How can we achieve these three standards? We strategized such a prototype: constant temperature amplification instead of PCR reaction to achieve convenience, targeted nicked DNA to achieve specificity, and selection of reporters that can be amplified to achieve high sensitivity.</p>
                            <p>On the <a href="https://2021.igem.org/Team:NEFU_China/Design" class="hyperlink">[Design]</a> page, you may find that we have selected recombinase polymerase amplification, nCas9 and Nb.BbvCI nickase, Phi29 DNA polymerase and Klenow Fragment, and a visual reporter, G-quadruplex. On this page, we will focus on the conceptual development and engineering cycle of the entire project, and present the progress of our engineering in an order of "design", "build", "test" and "learn", which is different from that of the <a href="https://2021.igem.org/Team:NEFU_China/Results" class="hyperlink">[result]</a> .</p>
                            <p>In addition, for the comprehensive development of our detection system, improving the specificity, sensitivity and simplicity of the detection is still essential. For this reason, we have also made efforts. We hope that any visitor can understand our conceptual process by browsing this page, and then initiate your own thinking.</p>
                        </div>
                        <div id="slide1-chapter2">
                            <h2>Design</h2>
                            <p>In our detection platform, we first need to amplify the nucleic acid sequence to be detected. After consulting literature, we finally chose RPA, a relatively stable and simple amplification method. For nickase to generate nicked DNA and for polymerase with single-strand displacement activity, we chose two enzymes for each of them. Among these enzymes, nCas9 and BbvC are the nickases, and Phi29 and Klenow possess single-strand displacement functions. Then, we proceeded to the expression, purification and functional analyses of these four proteins. Among them, we selected the enzymes with the highest efficiency, and used them in our detection system. As the final step, we verified the color development based on the peroxidase activity generated by G-quadruplex and hemin association.</p>
                        </div>
                        <div id="slide1-chapter3">
                            <h2>Build</h2>
                            <p>In the Build and Protein Expression sections, through our tenacious attempts, we successfully constructed 8 vectors to express our desired protein! These proteins include Phi29-linker-nCas9, Phi29-SH3, nCas9-slig, BbvC R1-ZF, BbvC R2-Phi29, BbvC R2-slig, Klenow-SH3 and Klenow mut-SH3.</p>
                            <p>In the single-chain displacement section, we use the electrophoretic mobility shift assay (EMSA) to verify the experimental success in generating nicked DNA and displacement of single strand DNA.</p>
                        </div>
                        <div id="slide1-chapter4">
                            <h2>Test</h2>
                            <h1>Section 1</h1>
                            <p><strong>Turning the spearhead - the mosaic method replaces the fusion</strong> </p>
                            <p>Initially, we planned to express Phi29-linker-nCas9 fusion protein and then investigate its activity. However, due to the large molecular weight, we could not successfully express this fusion protein, despite our continuous attempts to optimize the conditions of induction. After communicating with other iGEM teams, we designed another strategy of producing two proteins, Phi29-SH3 and nCas9-Slig, which could be assembled together, instead of expressing the fusion protein. Meanwhile, we have individually conducted functional verification of the two recombinant proteins and found that they were functionally active. See the <a href="https://2021.igem.org/Team:NEFU_China/Results" class="hyperlink">[Result]</a> page for details</p>
                            <div class="figure-container auto-margin">
                                <img src="https://2021.igem.org/wiki/images/0/03/T--NEFU_China--RCA_ab_prot.png" alt="">
                            </div>
                            <h1>Section 2</h1>
                            <p><strong>Compare separately and choose the best</strong></p>
                            In this part of the project, we mainly describe the induction of nickase expression and the functional verification. Through reviewing relevant literature, we learned that nCas9 has better specificity in recognizing specific sequences than BbvC. However, when verifying its activity, we found that BbvC showed better efficiency in creating nicked DNA than nCas9. See the <a href="https://2021.igem.org/Team:NEFU_China/Results" class="hyperlink">[Result]</a> page for details. Based on our experimental data and literature review, we decided to choose BbvC as the nickase and assemble it with a zinc finger protein to improve the specificity of DNA site recognition.
                            <h1>Section 3</h1>
                            <p><strong>Plan tactics, multiple parallel experiments to ensure success</strong></p>
                            <p>In this part, we mainly describe the induced expression of single-strand displacement polymerases and their functional verification. By exploring relevant literature, we learned that Phi29 not only has a single-strand replacement function, but also possesses 3'-to-5' exonuclease activity. To compare the performance of Phi29 and Klenow, we carried out functional verification of single-strand replacement and DNA polymerization. To our surprise, the results showed that Phi29 worked more efficiently than Klenow. Therefore, through the interpretation of the experimental data, we finally chose Phi29 as our single-strand replacement polymerase, and individually expressed the two proteins ZF-BbvC R1 and BbvC R2-Phi29. This could allow us to conduct DNA nicking and programmed single-strand replacement consecutively. The idea of "individual expression and combinatorial verification" was also based on our experience in the previous experiment.</p>
                            <h1>Section 4</h1>
                            <p><strong>Avoid False Positives and Application of "Basic Principles of Testing"</strong></p>
                            <p>Color reaction is extremely important in our project. It shows our results through color changes at a glance, reflecting the superiority of our rational engineering.</p>
                            <p>In our initial design, we designed a circular template for the RCA reaction. As shown in the figure, please see the <a href="https://2021.igem.org/Team:NEFU_China/Results" class="hyperlink">[Result]</a> section for the detailed process</p>
                            <p>However, exonuclease digestion cannot completely remove the lock probe, resulting in RCA reactions and color development of the negative control group, although the degree of color development could be distinguished under laboratory conditions, such as quantitative measurement by a spectrophotometer. [Jump result]. However, in practice, this type of instrument may not be always available, and the color changes in negative controls may lead to false positive results. Therefore, we must carefully design our protocol to eliminate the possibility of false positive. Through extensive discussion and literature exploration, we abandoned the circular template, and started to test linear templates that could be cyclized in the presence of a lock oligonucleotide and T4 DNA ligase, and then used as a template in the RCA. In this design, the negative control group did not develop any detectable color.</p>
                            <div class="figure-container auto-margin">
                                <img src="https://2021.igem.org/wiki/images/0/03/T--NEFU_China--RCA_ab_prot.png" alt="">
                            </div>
                        </div>
                        <div id="slide1-chapter5">
                            <h2>Learn</h2>
                            <p>In synthetic biology, the design of gene expression strategies needs to take many factors into account. For example, proteins with large molecular weights are difficult to be expressed in E. coli. Therefore, our phi29-linked-ncas9 fusion protein could not be well expressed in bacteria, as we hoped. </p>
                            <p>The strategy of expressing large size proteins separately and then assemble them together to perform their functions provided ideas for the subsequent expression of many proteins. If the objective laws are not well observed, it may waste considerable manpower and material resources but reach unsatisfied results.</p>
                            <p>In our project, the actual feasibility is a matter of principle. Our purpose is to improve the human society, and thus we should not leave any chance for false positive results, as mentioned above. Since it is a matter of principle, we must adjust our strategy to circumvent it.</p>
                        </div>
                        <div id="slide1-chapter6">
                            <h2>Outlook</h2>
                            <p>With the development of molecular biology and synthetic biology, our detection platform will also face increasing challenges. It is very important to further improve the accuracy and specificity of our detection method. We are considering to replace the Cas9 by Cas12a or Cas12b, which will reduce potential off-target effects for better accuracy and utilize smaller nickases for easier test operation. We will keep our active and innovative research development, and please continue to pay attention to our future studies!</p>
                        </div>
                    </div>
                </div>
            </section>
        </div>
    </div>
</div>
<a href="#" id="mobile-to-top-arrow" class="to-top">
    <p style="font-size: 10px;text-align: center;">Top of</p>
    <p style="font-size: 10px;text-align: center;">the page</p>
    <img style="max-height: 70px" src="../../resources/to_top.png">
</a>
<!---------- FOOTER ------------>
<footer id="footer">
    <div class="contact-info">
        <div class="contact-info-content">
            <!--- About Us-->
            <div class="box">
                <h3>
                    About Us
                </h3>
                <p>
                    This team is composed of enthusiastic  students from three different colleges and various majors. To spy out the secrets of directed evolution, a group of beginners are looking forward to creating extraordinary experiences and we did it.
                </p>
            </div>
            <!---- Follow Us -->
            <div class="box">
                <h3>
                    Helpful links
                </h3>
                <h4><a target="_blank" href="../../resources/poster.jpg">Poster</a></h4>
                <h4><a target="_blank" href="../../page/PROJECT/Design.html">Design</a></h4>
                <h4><a target="_blank" href="../../page/DOCUMENTATION/Notebook.html">Notebook</a></h4>
            </div>
            <!---- Contact Us-->
            <div class="box">
                <h3>
                    Address
                </h3>
                <p>
                    Northeast Forestry University, Harbin, Heilongjiang 150040, P. R. China
                </p>
            </div>
        </div>
        <hr color="white">
        <div class="contact-info-content">
            <div class="contact-icon">
                <a target="_blank" href=""> <i class="fa fa-envelope"></i>IDEC_NEFU2021@163.com</a>
            </div>
            <div class="contact-icon">
                <a target="_blank" href=""> <i class="fab fa-facebook-square"></i>iDEC NEFU China</a>
            </div>
            <div class="contact-icon">
                <a target="_blank" href=""> <i class="fab fa-twitter-square"></i>iDEC NEFU</a>
            </div>
        </div>
    </div>
</footer>

<script src= "../../JS/jQuery.js" type =text/javascript></script>
<script src="../../JS/detect.js" type= "text/javascript"></script>
<script src="../../JS/main_javascript.js" type =text/javascript></script>
<script src="../../JS/MoveContent.js" type= "text/javascript"></script>
<script src="https://kit.fontawesome.com/50c1a3d91f.js" crossorigin="anonymous"></script>

<script>
    document.addEventListener("DOMContentLoaded", function(event) {
        MoveContentOutside();
    });
</script>
</body>
</html>
