<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>iDEC NEFU_China 2021</title>
  <link rel="stylesheet" type="text/css" href="../../CSS/main_cssstyle.css">
  <style>
    header{
      background: url("../../resources/header1.png");
      min-height: 450px;
      background-repeat: no-repeat;
      background-size: cover;
    }
    .page-title {
      padding-top:10em;
      padding-bottom: 1.5em;
    }
    /* footer的样式 */
    .contact-icon{
      display: flex;
      align-items: center;
      /*font-size: 3em;*/
      color: white;
    }
    .contact-icon a{
      color: white;
    }
    .contact-icon i{
      font-size: 3em;
    }
  </style>
</head>
<body>

<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
<header>
  <!------------ 页面标题PAGE TITLE -------------------->
  <div class="page-title">
    <div class="page-title-content">
      <h1>Design</h1><!---Page title -->
    </div>
  </div>
</header>
<div class="grid-normal">
  <!--------------Introduction/Summary--------------------->
  <div class="introduction grid-normal">
  </div>
</div>
<div style="display:none" id="marker" class= "triangle-down"> </div>
<!------------------ BUTTONS ---------------------------->
<div class="buttons">
</div>
<div class="sections-container">
  <aside class="sticky">
    <div id="sub-nav"class="sub-navigation-content">
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter1">Overview</a><!------Chapter one -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter2">RPA</a><!------Chapter Two -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter3">Targeted Nicking</a><!------Chapter Three -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter4">Single-stranded DNA Displacing</a><!------Chapter Three -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter5">RCA</a><!------Chapter Three -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter6">G4 Visualization</a><!------Chapter Three -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter7">Conclusion</a><!------Chapter Three -->
      </div>
      <div class="sub-navigation-chapter cool-link2">
        <a class= "main-nav-link" linkedidchapter="slide1-chapter8">References</a><!------Chapter Three -->
      </div>
    </div>
  </aside>
  <div class="carousel">
    <div class="slider">
      <!------------------------------ section 1------------------------------------>
      <section class="slide-sections">
        <div class="main-body">
          <div class="main-body-content">
            <div id="slide1-chapter1">
              <h2>Overview</h2>
              <p>Due to the biohazard potential of infectious viruses, such as SARS-CoV-2, and their nucleotide sequences, it is rational to avoid using any of the viruses or their sequences in our research project. Therefore, we want to establish the detection system using the M13 phage <span style="font-style: italic"> geneIII </span> to test the concept. The detection signal can be accelerated through the recombinase polymerase amplification (RPA) technology that specifically and exponentially amplifies the <span style="font-style: italic"> geneIII </span>. Then, the resultant double-stranded DNA forms a specifically nicked site under the targeted nicking system. A special DNA polymerase continues to synthesize nucleotides in 5'-to-3' direction from the nicked site, thus replacing a short single-stranded DNA (ssDNA), which previously hybridizes its complementary strand in the double strand DNA.</p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/e/ec/T--NEFU_China--Design-1.png" alt="">
              </div>
              <p>The displaced ssDNA enters the rolling circle amplification (RCA) as a primer. The template for the RCA is a complementary sequence that also contains G-quadruplex (G4) motifs. Multiple G4 motifs produce through the RCA bind hemin that acts as an activated peroxidase to oxidize a substrate, 2,2'-azinobis-3-ethylbenzothiazoline-6-sulfonate (ABTS). The color’s change in the oxidation process can be visualized very quickly, which provides an instant detection of a virus.</p>
            </div>
            <div id="slide1-chapter2">
              <h2>RPA</h2>
              <p>Recombinase polymerase amplification (RPA) is an isothermal amplification method based on special enzymes. The RPA system contains three core components, including a recombinase, a single-stranded DNA binding protein and a DNA polymerase. <a linkedidchapter="r1" class="main-nav-link hyperlink"><sup>[1]</sup></a>  The recombinase binds to a primer to form a nucleoprotein complex that scans double-stranded DNA and facilitate strand exchange at its cognate sites. The resulting D-loop structure is stabilized by the single-stranded DNA binding protein interacting with the displaced template strand. Then, the recombinase disassembles and leaves the primer. The DNA polymerase synthesizes the complementary strand by adding nucleotides to the 3’-end of the primer. </p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/b/ba/T--NEFU_China--Design-2.png" alt="">
              </div>
              <p>We designed a pair of primers specific to the <span style="font-style: italic"> geneIII </span> of M13 phage. If the tested sample contains the sequence of the <span style="font-style: italic"> geneIII </span>, a specific and exponential replication can be initiated to achieve signal amplification using our detecting system. </p>

            </div>
            <div id="slide1-chapter3">
              <h2>Targeted Nicking</h2>
              <p>In order to create nicks at the specific sites of double-stranded DNA, we explored different strategies and were inspired by the development of gene editing techniques. Three generations of nucleases for targeted gene editing have been developed, including the zinc-finger nuclease (ZFNs), the transcriptional activation-like effector nuclease (TALEN) and the CRISPR/Cas9 system. <a linkedidchapter="r2" class="main-nav-link hyperlink"><sup>[2-3]</sup></a> </p>
              <p>We chose both ZFNs and CRISPR/Cas9 systems to make parallel and comparative designing and studies, to find an optimal targeted nicking system.</p>
              <h1>sgRNA-nCas9</h1>
              <p>In the CRISPR/Cas9 system, Cas9 undergoes conformational changes upon sgRNA binding, and is subsequently directed to its target site. Then, Cas9 introduces a double strand break (DSB) in the DNA sequence of the target site. <a linkedidchapter="r4" class="main-nav-link hyperlink"><sup>[4]</sup></a>
                Cas9 has two nuclease domains: the HNH and RuvC-like domains. The Cas9 HNH nuclease domain cleaves the complementary strand, while the Cas9 RuvC-like domain cleaves the template strand. <a linkedidchapter="r5" class="main-nav-link hyperlink"><sup>[5]</sup></a>
                In our project, we created a HNH nuclease activity-null mutant (H840A), designated as nCas9, which can be used to generate DNA break in the template strand.
              </p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/0/00/T--NEFU_China--Design-3.png" alt="">
              </div>
              <h1>ZF-Nb.BbvCI</h1>
              <p>The restriction endonuclease BbvCI from <span style="font-style: italic"> Bacillus brevis </span>  recognizes a non-palindromic sequence CC^TCAGC/GC^TGAGG. BbvCI consists of two heterogeneous subunits, R1 and R2. <a linkedidchapter="r6" class="main-nav-link hyperlink"><sup>[6]</sup></a>
                The R1 subunit cleaves the bottom chain (GC^TGAGG, where ^ indicating the cleavage site), while the R2 subunit cuts the top chain (CC^TCAGC). The inactivating mutations of R1 or R2 do not disrupt protein folding and DNA sequence recognition of BbvCI. <a linkedidchapter="r7" class="main-nav-link hyperlink"><sup>[7]</sup></a>
              </p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/9/90/T--NEFU_China--Deisgn-4.png" alt="">
              </div>
              <p>In our project, we used the R1+R2- mutant, named as Nb.BbvCI. In addition, we explored the Zinc Finger Tools (URL http://www.zincfingertools.org.) created by <span style="font-style: italic"> Jeffrey G. Mandell et al </span>. <a linkedidchapter="r8" class="main-nav-link hyperlink"><sup>[8]</sup></a>
                Using the experimentally characterized database of zinc finger domains, the expected amino acid sequence of a zinc finger protein bound to the selected target can be generated. The zinc finger protein can meet the needs of specific identification to a certain extent.
              </p>
            </div>
            <div id="slide1-chapter4">
              <h2>Single-stranded DNA Displacing</h2>
              <p>For the purpose to obtain ssDNA, we added Phi29 DNA polymerase to the system, which is an enzyme widely used in RCA. Phi29 catalyzes 5’-to-3’ polymerization and possesses strand displacement activity, which enables it to displace the complementary strand in double-stranded regions during DNA synthesis. <a linkedidchapter="r9" class="main-nav-link hyperlink"><sup>[9]</sup></a>
                In our design, Phi29 continues to add nucleotides from the nick site. During this elongation process, the complementary strand behind the gap at 3'-side can be displaced. The resultant short ssDNA will act as a primer in the RCA reaction.
              </p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/2/24/T--NEFU_China--Deisgn-5.png" alt="">
              </div>
              <p>After extensively exploring literature, we found that the 3’-to-5’ ssDNA exonucleolytic activity of Phi29 could adversely affect the results by potentially degrading the displaced ssDNA. <a linkedidchapter="r10" class="main-nav-link hyperlink"><sup>[10]</sup></a>
                Therefore, we decided to test Klenow, the large fragment of DNA polymerase I, in comparison to Phi29, and evaluate the efficiency of the ssDNA production.
              </p>
            </div>
            <div id="slide1-chapter5">
              <h2>RCA</h2>
              <p>Rolling circle amplification (RCA) offers a simple method for DNA amplification, which is an isothermal DNA amplification without a thermal cycling reactor. Thus, it is more suitable to field-test than the methods depending on a thermo cycler. <a linkedidchapter="r11" class="main-nav-link hyperlink"><sup>[11]</sup></a></p>
              <p>The displaced ssDNA, complementary to the template sequence, can bridge the nicked site between the 5’- and 3’-termini of the template strand through base pairing. With the help of this displaced ssDNA, the nick at the template can be sealed by adding T4 DNA ligase, which catalyzes phosphodiester bond formation between juxtaposed 5'-phosphate and 3'-hydroxyl termini. </p>
              <p>We can skip the primer removal and recombination process, and directly put the ssDNA-template hybrid mix for RCA, which simplifies the procedure and saves possible extra cost of ssDNA-specific exonuclease, such as exonucleaseI. <a linkedidchapter="r9" class="main-nav-link hyperlink"><sup>[9]</sup></a></p>
              <div class="figure-container auto-margin">
                <img src="https://2021.igem.org/wiki/images/a/a2/T--NEFU_China--Deisgn-6.png" alt="">
              </div>
              <p>In the RCA reaction, Phi29 continues to synthesize ssDNA from the ssDNA (as a primer) with a high processivity and strong DNA strand displacement activity. The resultant long chain ssDNA is composed of the tandem repeat of G-tract (G ≥ 3) sequences, which can form G-quadruplex needed in the following test. We also used Klenow to replace the role of Phi29 in RCA and evaluated whether it could possibly improve the amplification yield.</p>

            </div>
            <div id="slide1-chapter6">
              <h2>G4 Visualization</h2>
              <p>G4 is a highly stable DNA secondary structure formed by single-stranded G-rich nucleic acids. This structure can associate  with a cofactor, hemin, to produce catalytic activity similar to that of a peroxidase, which is thereby designated as deoxygenase (DNAzyme). <a linkedidchapter="r12" class="main-nav-link hyperlink"><sup>[12]</sup></a>
                We used 2,2'-azinobis ABTS as a substrate to analyze the G4 levels produced in the RCA reaction. In the presence of H<sub>2</sub>O<sub>2</sub>, the ABTS<sup>2-</sup>is oxidized by the complex to produce the colored radical anion (ABTS-). The visible color change can be further quantified in a spectrophotometer at the wavelength of 420 nm.
              </p>
            </div>
            <div id="slide1-chapter7">
              <h2>Conclusion</h2>
              <p>In practical application, RNA is easy to be degraded and its preparation process is relatively difficult. Besides, we discovered that the nick-generating activity of Nb.BbvCI is better than that of nCas9. Therefore, we decided to use ZF-Nb.BbvCI in our detection system. Meanwhile, after comprehensively analyzing the capability of the single chain displacement and RCA using Phi29 and Klenow, we chose the former one to be used in our system. <a href="https://2021.igem.org/Team:NEFU_China/Results" class="hyperlink">[Results]</a>
                Furthermore, we generated two fusion proteins by connecting the R1 and R2 domains of Nb.BbvCI to the ZF and Phi29, respectively. This allowed us to produce their quaternary protein complex through <span style="font-style: italic"> in vitro </span> assembly, which could generate an all-in-one reaction module with combined nick-creating, displacement and amplification.
              </p>
            </div>

            <div class="references" id="slide1-chapter8">
              <h2>References</h2>
              <ol>
                <li id="r1">Piepenburg, Olaf et al. “DNA detection using recombination proteins.” PLoS biology vol. 4,7 (2006): e204. doi:10.1371/journal.pbio.0040204</li>
                <li id="r2">Esvelt, Kevin M, and Harris H Wang. “Genome-scale engineering for systems and synthetic biology.” Molecular systems biology vol. 9 (2013): 641. doi:10.1038/msb.2012.66</li>
                <li>Puchta, Holger, and Friedrich Fauser. “Gene targeting in plants: 25 years later.” The International journal of developmental biology vol. 57,6-8 (2013): 629-37. doi:10.1387/ijdb.130194hp</li>
                <li id="r4">Zhan, Tianzuo et al. “CRISPR/Cas9 for cancer research and therapy.” Seminars in cancer biology vol. 55 (2019): 106-119. doi:10.1016/j.semcancer.2018.04.001</li>
                <li id="r5">Chuang, Chin-Kai, and Wei-Ming Lin. “Points of View on the Tools for Genome/Gene Editing.” International journal of molecular sciences vol. 22,18 9872. 13 Sep. 2021, doi:10.3390/ijms22189872</li>
                <li id="r6">Shen, Betty W et al. “Structure, subunit organization and behavior of the asymmetric Type IIT restriction endonuclease BbvCI.” Nucleic acids research vol. 47,1 (2019): 450-467. doi:10.1093/nar/gky1059</li>
                <li id="r7">Heiter, Daniel F et al. “Site-specific DNA-nicking mutants of the heterodimeric restriction endonuclease R.BbvCI.” Journal of molecular biology vol. 348,3 (2005): 631-40. doi:10.1016/j.jmb.2005.02.034</li>
                <li id="r8">Mandell, Jeffrey G, and Carlos F Barbas 3rd. “Zinc Finger Tools: custom DNA-binding domains for transcription factors and nucleases.” Nucleic acids research vol. 34,Web Server issue (2006): W516-23. doi:10.1093/nar/gkl209</li>
                <li id="r9">Kobori, Toshiro, and Hirokazu Takahashi. “Expanding possibilities of rolling circle amplification as a biosensing platform.” Analytical sciences : the international journal of the Japan Society for Analytical Chemistry vol. 30,1 (2014): 59-64. doi:10.2116/analsci.30.59</li>
                <li id="r10">Johne, Reimar et al. “Rolling-circle amplification of viral DNA genomes using phi29 polymerase.” Trends in microbiology vol. 17,5 (2009): 205-11. doi:10.1016/j.tim.2009.02.004</li>
                <li id="r11">Jiang, Hong-Xin et al. “G-quadruplex fluorescent probe-mediated real-time rolling circle amplification strategy for highly sensitive microRNA detection.” Analytica chimica acta vol. 943 (2016): 114-122. doi:10.1016/j.aca.2016.09.019</li>
                <li id="r12">Stefan, Loic et al. “Insights into how nucleotide supplements enhance the peroxidase-mimicking DNAzyme activity of the G-quadruplex/hemin system.” Nucleic acids research vol. 40,17 (2012): 8759-72. doi:10.1093/nar/gks581</li>
              </ol>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>
</div>
<a href="#" id="mobile-to-top-arrow" class="to-top">
  <p style="font-size: 10px;text-align: center;">Top of</p>
  <p style="font-size: 10px;text-align: center;">the page</p>
  <img style="max-height: 70px" src="../../resources/to_top.png">
</a>
<!---------- FOOTER ------------>
<footer id="footer">
  <div class="contact-info">
    <div class="contact-info-content">
      <!--- About Us-->
      <div class="box">
        <h3>
          About Us
        </h3>
        <p>
          This team is composed of enthusiastic  students from three different colleges and various majors. To spy out the secrets of directed evolution, a group of beginners are looking forward to creating extraordinary experiences and we did it.
        </p>
      </div>
      <!---- Follow Us -->
      <div class="box">
        <h3>
          Helpful links
        </h3>
        <h4><a target="_blank" href="../../resources/poster.jpg">Poster</a></h4>
        <h4><a target="_blank" href="../../page/PROJECT/Design.html">Design</a></h4>
        <h4><a target="_blank" href="../../page/DOCUMENTATION/Notebook.html">Notebook</a></h4>
      </div>
      <!---- Contact Us-->
      <div class="box">
        <h3>
          Address
        </h3>
        <p>
          Northeast Forestry University, Harbin, Heilongjiang 150040, P. R. China
        </p>
      </div>
    </div>
    <hr color="white">
    <div class="contact-info-content">
      <div class="contact-icon">
        <a target="_blank" href=""> <i class="fa fa-envelope"></i>IDEC_NEFU2021@163.com</a>
      </div>
      <div class="contact-icon">
        <a target="_blank" href=""> <i class="fab fa-facebook-square"></i>iDEC NEFU China</a>
      </div>
      <div class="contact-icon">
        <a target="_blank" href=""> <i class="fab fa-twitter-square"></i>iDEC NEFU</a>
      </div>
    </div>
  </div>
</footer>

<script src= "../../JS/jQuery.js" type =text/javascript></script>
<script src="../../JS/detect.js" type= "text/javascript"></script>
<script src="../../JS/main_javascript.js" type =text/javascript></script>
<script src="../../JS/MoveContent.js" type= "text/javascript"></script>
<script src="https://kit.fontawesome.com/50c1a3d91f.js" crossorigin="anonymous"></script>

<script>
  document.addEventListener("DOMContentLoaded", function(event) {
    MoveContentOutside();
  });
</script>
</body>
</html>
